client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -95,4 +123,4 @@ const mapStateToProps = state => {
 	return { status };
 };
 
-export default connect( mapStateToProps )( localize( PluginAutomatedTransfer ) );
+export default connect( mapStateToProps )( localize( wrapWithClickOutside( PluginAutomatedTransfer ) ) );
```

Comment:
I think we need to set the `pure` option to false since this component now has its own state: https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments